generated/Endpoint/ChatPostMessage.php

```diff
@@ -17,6 +17,8 @@ class ChatPostMessage extends \JoliCode\Slack\Api\Runtime\Client\BaseEndpoint im
 {
     use \JoliCode\Slack\Api\Runtime\Client\EndpointTrait;
 
+    protected array $formParameters;
```

Comment:
Sadly this file cannot be edited, it's generated by JanePHP.

Reply:
As far as I can tell we are already using the latest Jane 7.5 version so I will create an issue in that repo 👍🏻

Reply:
so... its been over a month and no one seems to care in the JanePHP repo about this issue...